php/util.php

```diff
@@ -5,8 +5,15 @@
 require_once( $rootPath.'/conf/config.php' );
 
 // Automatically include only the used utility classes
-spl_autoload_register(function ($class) {
-    require_once 'utility/'. strtolower($class). '.php';
+spl_autoload_register(function ($class) 
+{
+	// Remove namespaces from the classname string
+	// Important for compatibility with 3rd party plugins
+	if (preg_match('@\\\\([\w]+)$@', $class, $matches)) {
```

Comment:
We really need PCRE here? What about
```
	$arr = explode('\\',$class);
	$class = end($arr);
```
?
IMHO this will be faster.

Reply:
Alright done.